examples/config/cosmodc2_config.yml

```diff
@@ -101,8 +115,21 @@ TXTwoPoint:
     max_sep: 250
     nbins: 20
     verbose: 0
+    shape_noise: True
 
-
+TXTwoPointLensCat:
```

Comment:
Is this related to this PR?

Reply:
When using `cosmodc2_redmagic_pipeline.yml`, it requires to have that stage defined I think so I added this there to fix it.